lava-v2-jobs-from-api.py

```diff
@@ -270,7 +270,7 @@ def jinja_render(job):
     parser = argparse.ArgumentParser()
     parser.add_argument("--token", help="KernelCI API Token")
     parser.add_argument("--api", help="KernelCI API URL")
-    parser.add_argument("--storage", help="KernelCI storage URL")
+    parser.add_argument("--storage", help="KernelCI storage URL", required=True)
```

Comment:
It's not required on the command line as it can also be found in the config file.  Instead, a check should be done in the main function, for example following the other required config options:

```
    if not token:
        raise Exception("No token provided")
    if not api:
        raise Exception("No KernelCI API URL provided")
    if not storage:
        raise Exception("No KernelCI storage URL provided")
```